libs/adapters/tsconfig.json

```diff
@@ -1,11 +1,13 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "rootDir": "src",
-    "outDir": "build",
+    "rootDir": ".",
     "tsBuildInfoFile": "build/.tsbuildinfo",
```

Comment:
tsBuildInfoFile is also a build thing right?

Reply:
Removing this from the base tsconfig is only possible if we also remove the references. As described in the issue above, this isn't possible without some concessions.